compiler/resolution/callDestructors.cpp

```diff
@@ -552,6 +552,8 @@ void ReturnByRef::transformMove(CallExpr* moveExpr)
   Symbol*   useLhs   = toSymExpr(lhs)->symbol();
   Symbol*   refVar   = newTemp("ret_to_arg_ref_tmp_", useLhs->type->refType);
 
+  FnSymbol* unaliasFn = getUnalias(useLhs->type);
```

Comment:
getUnalias may be run more than necessary here; I'd only set unaliasFn in the body of the conditional below with the comment and typesOK. Or just compute it twice.